### DISCUSSION TITLE
[PRO] EasyMDE icons not showing?

### DISCUSSION THREAD

**Discussion by demogar** [Laravel-Backpack/community-forum#778] [Q&A (Help)] [answered]
When publishing to a `staging` environment, I notice some of the fonts are not loading:

```
downloadable font: download failed (font-family: "Font Awesome 6 Free" style:normal weight:900 stretch:100 src index:0): status=2147746065 source: https://.../storage/basset/cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/webfonts/fa-solid-900.woff2
downloadable font: download failed (font-family: "Font Awesome 6 Free" style:normal weight:900 stretch:100 src index:1): status=2147746065 source: https://.../storage/basset/cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/webfonts/fa-solid-900.ttf
```

The url looks good, but the `assets` are not present in that directory, so my guess is that they never got included through `basset`:

```
@basset('https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/webfonts/fa-solid-900.woff2', false)
@basset('https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/webfonts/fa-brands-400.woff2', false)
@basset('https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/webfonts/fa-regular-400.woff2', false)
```

Currently installed versions:

```
"backpack/crud": "^6.3", // 6.4
"backpack/pro": "^2.0", // 2.0.18
```

**Comment by phpfour** [comment]
Hi @demogar,

Thanks for posting your question. I just checked and found that the following three fonts are loaded for the icon picker field: 

```
@basset('https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/webfonts/fa-solid-900.woff2', false)
@basset('https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/webfonts/fa-brands-400.woff2', false)
@basset('https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/webfonts/fa-regular-400.woff2', false)
```

For the EasyMDE field, the following CSS is loaded: 

```
@basset('https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css')
@basset('https://unpkg.com/easymde@2.18.0/dist/easymde.min.css')
```

Anyway, I will do a thorough check and get back to you if I can reproduce the issue. 

Cheers

**Reply by demogar** [reply]
Hey @phpfour , thanks for your response.

Wouldn't that mean that, in order to show the icons, it would be necessary to show a `icon picker` field?

I'm unable to get this running.  I had add the lines manually to get it working once deployed.

**Reply by promatik** [reply]
Hey @demogar 🙌
I just pushed a fix for this, it will soon be merged and a `composer update` will solve the issue for you.

Once it gets merged, we update this thread 🙌

**Reply by pxpm** [reply, marked as answer]
So.. the fix was merged. Sorry for the delay, I let this one slip through the cracks. 

You can get it with backpack/pro 2.0.23. Please allow a few moments from this comment in time for our private repository to pick the released code and allow the update with `composer update`. 

Cheers
